mysql-test/suite/galera/r/galera_ist_mariabackup.test

```diff
@@ -0,0 +1,10 @@
+--source include/big_test.inc
+--source include/galera_cluster.inc
+--source include/have_innodb.inc
+--source include/have_mariabackup.inc
+
+--source suite/galera/include/galera_st_disconnect_slave.inc
+--source suite/galera/include/galera_st_shutdown_slave.inc
+
+--source suite/galera/include/galera_st_kill_slave.inc
+--source suite/galera/include/galera_st_kill_slave_ddl.inc
```

Comment:
This include file have "if ($have_debug)" so there is different result for debug build and release build. Record result using release build and then create .rdiff file using debug build. See instructions how to do that from: https://mariadb.com/kb/en/library/mysql-test-auxiliary-files/

Reply:
File name is then something like r/galera_ist_mariabackup,debug.rdiff